src/Symfony/Component/DependencyInjection/Compiler/AutowirePass.php

```diff
@@ -130,7 +130,6 @@ private function doProcessValue($value, $isRoot = false)
             return $value;
         }
 
-        $autowiredMethods = $this->getMethodsToAutowire($reflectionClass);
```

Comment:
The `getMethodsToAutowire` still exists in this class, can be removed now

Reply:
indeed, done, thanks